example/src/App.tsx

```diff
@@ -4,11 +4,11 @@ import { useLazerpay } from './dist/index'
 
 const App = () => {
   const config = {
-    publicKey: 'pk_live_0N24k7lsrr7NGfrDQpIjPGy9z61LkXjUqxX3r99XblXHemwMht',
+    publicKey: 'pk_live_0N24k7lsrr7NGfrDQpIjPGy9z61LkXjUqxX3r99XblXHemwMht', // live key
```

Comment:
Can we put this in .env instead?

Reply:
uhmm do you think it's really necessary? This is meant to be just an example app (file) anyone can look at

Reply:
Ok ok